src/components/liveSession/services/liveSessionUi.service.js

```diff
@@ -83,18 +83,43 @@
                 _endLiveSession();
             }
 
-            function showStudentLiveSessionPopUp(){
-                var translationsPromMap = {};
+            function showStudentConfirmationPopUp(){
+                let translationsPromMap = {};
                 translationsPromMap.title = $translate('LIVE_SESSION.LIVE_SESSION_REQUEST');
                 translationsPromMap.content= $translate('LIVE_SESSION.WANT_TO_JOIN');
-                translationsPromMap.acceptBtnTitle = $translate('LIVE_SESSION.REJECT');
-                translationsPromMap.cancelBtnTitle = $translate('LIVE_SESSION.ACCEPT');
+                translationsPromMap.acceptBtnTitle = $translate('LIVE_SESSION.JOIN');
+                translationsPromMap.cancelBtnTitle = $translate('LIVE_SESSION.CANCEL');
```

Comment:
Should be "Decline"
(not cancel).

See other comments in en.json

Reply:
done